### ISSUE TITLE
Can't create an `AttributeType` because it's an enum

### ISSUE THREAD

**Issue by aSemy** [lgwillmore/warden#39]
I'm following the docs, but I can't see how to make a new instance of `AttributeType`, because it's an enum

https://warden-kotlin.netlify.app/attributes/#hasattributes-and-attributetype

```kotlin
val USER_TYPE = AttributeType(
    type = "USER",
    typeKeyword = "authType"
)
```

https://github.com/lgwillmore/warden/blob/e3e984ccad746e0615cfb347002ad2d6b183f4d0/core/src/commonMain/kotlin/codes.laurence.warden/policy/expression/policies.kt#L102-L107

**Comment by aSemy** [comment]
I also can't access `HasAttributesI`. 

I think a step was missed when the 'attributes' module was split into a different module. It's not available, or exposed as an API dependency from the core module. I can't see what the artifact is called because the jfrog Maven repo doesn't seem to allow navigating the directory structure.

**Comment by lgwillmore** [comment]
Sorry, no reliable internet till Monday. New module artifact is 'warden-atts'.

**Comment by lgwillmore** [comment]
Fixed by pr #41 . Documentation also updated for installation of dependency. Available in release 0.3.0

**Comment by lgwillmore** [comment]
Hey, sorry. Not on reliable internet till Monday. Yes, spilt out to
'warden-atts' artifact. Will add the import installation docs.
